addons/website/static/src/js/website.contentMenu.js

```diff
@@ -236,6 +238,7 @@ var EditMenuDialog = widget.Dialog.extend({
     add_menu: function () {
         var self = this;
         var dialog = new MenuEntryDialog(this, {menu_link_options: true}, undefined, {});
+        dialog.$modal.addClass('o_contentMenu_link_modal');
```

Comment:
@qsm-odoo 👍  ok, I agree... how do you want to proceed?
Should I create another task and made a proposal?
Or we just create a task and put it in hold waiting for websitepocalypse?